### PR TITLE
Use Accept-Language header in FAQ queries

### PR DESCRIPTION
**Pull request by jaroslav-kubicek**
use `Accept-Language` in all FAQ dataloaders

### CODE REVIEW

File: src/FAQ/dataloaders/getFAQCategory.js [hunk 1]
```diff
@@ -15,16 +16,15 @@ type RawFAQArticleItem = {
 
 export type FAQArticleItem = RawFAQArticleItem & {
   id: string,
-  language: string,
 };
 
-export type Args = {|
-  language: string,
-|};
+export type Args = {||};
```
Comment (mrtnzlml) [inline src/FAQ/dataloaders/getFAQCategory.js]: @jaroslav-kubicek This type looks so odd. Why it's needed after this change? 🙂

**Comment by mrtnzlml** [comment]
@jaroslav-kubicek Also, please rebase it on top of the master branch... :)

File: src/FAQ/dataloaders/getFAQCategory.js (hunk 1 above)
Comment (jaroslav-kubicek) [reply src/FAQ/dataloaders/getFAQCategory.js]: It's for `Dataloader<Args, Output>` type

**Comment by mrtnzlml** [comment]
@jaroslav-kubicek I am very sorry, could you please rebase again? There is a lot of Flow errors:

```
...

Error: src/FAQ/mutations/addFAQArticleComment.js:51
 51:       ? LocaleMap[locale]
                       ^^^^^^ property `no_NO`. Property not found in
 51:       ? LocaleMap[locale]
             ^^^^^^^^^ object literal


... 17 more errors (only 50 out of 67 errors displayed)

...

**Comment by jaroslav-kubicek** [comment]
rebased

btw it failed very correctly in fact. Although it doesn't like much assigning to `$Keys<typeof ISOLocalesToObsolete>`, it correctly recognized mutation `addFAQArticleComment` was working with incorrect locales 😌 🌈 🦄

File: src/FAQ/queries/AllFAQCategories.js [hunk 2]
```diff
@@ -27,9 +27,7 @@ export default {
     { language, ...args }: Object,
     { dataLoader }: GraphqlContextType,
   ) => {
-    const results = await dataLoader.FAQCategories.load({
-      language,
-    });
+    const results = await dataLoader.FAQCategories.load({});
```
Comment (mrtnzlml) [inline src/FAQ/queries/AllFAQCategories.js]: This is really pissing me off. The fact that you have to pass an empty object in order to make it work... 😕

**Pull request merged**